machine_management/applying-autoscaling.adoc

```diff
@@ -30,13 +30,24 @@ include::modules/cluster-autoscaler-cr.adoc[leveloffset=+2]
 :FeatureName: ClusterAutoscaler
 include::modules/deploying-resource.adoc[leveloffset=+2]
 
+.Next steps
+
+* After you configure the ClusterAutoscaler, you must configure at least one
+MachineAutoscaler.
+
 [id="configuring-machineautoscaler"]
 == Configuring the MachineAutoscalers
 
-After you deploy the ClusterAutoscaler, you can
+After you deploy the ClusterAutoscaler,
```

Comment:
Not sure what we're trying to convey here. Maybe:

- After you deploy the ClusterAutoscaler, deploy MachineAutoscaler resources referencing the MachineSet that will be used to scale the cluster.

Dunno - opinions? @bison

Reply:
Yeah, that might be a bit clearer.